charts/linkerd-control-plane/values.yaml

```diff
@@ -290,6 +290,8 @@ controllerImage: cr.l5d.io/linkerd/controller
 controllerReplicas: 1
 # -- User ID for the control plane components
 controllerUID: 2103
+# -- Optionally allow a specific container image Tag (or SHA) to be specified for the controllerImage.
+controllerImageVersion: ""
```

Comment:
Can you move this up, right below `controllerImage`?

Reply:
No problem. Done.